src/index.js

```diff
@@ -0,0 +1,15 @@
+import * as EPSG_2056 from './EPSG_2056';
```

Comment:
we could have something like:
```js
export {default as EPSG_2056} from './EPSG_2056';
```
instead of the `import` followed by the `export`.
Untested, based on the how it's done in OpenLayers:
https://github.com/openlayers/openlayers/blob/master/src/ol/control.js

Reply:
No, unfortunately I think this doesn't work (yet), because here we have to export everything, not just the default export. See last comment [here](https://stackoverflow.com/a/38077264):

> you cannot currently export as named namespaces like that. See github.com/tc39/proposal-export-ns-from for the proposal for it. One thing you can do in the meantime is something like: ` import * as Something from './something'; export { Something } `

Reply:
And what about:
```
export {default as EPSG_2056_code} from './EPSG_2056';
export {* as EPSG_2056} from './EPSG_2056';
```

Reply:
Nope, `export * as something` does not exist yet, it's just [a proposal](https://github.com/tc39/proposal-export-ns-from).

Reply:
No, it's not working with microbundle, nor probably with TypeScript or any other tool to work with ES6 modules.

Reply:
OK, thanks for the test :-)